packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -446,6 +446,7 @@ class ThemeData {
   static ThemeData lerp(ThemeData begin, ThemeData end, double t) {
     assert(begin != null);
     assert(end != null);
+    assert(t != null);
```

Comment:
if you're gonna do this one, you should add it to all the `lerp` functions across the framework (and maybe even those in the engine), as well as the `lerp` methods on the Tween subclasses. Maybe do that in a separate PR though...